package.json

```
@@ -13,7 +13,8 @@
   "private": true,
   "scripts": {
     "sassWatch": "sass --watch src/assets/styles:dist/assets/styles",
-    "sassBuild": "sass src/assets/styles/main.scss:dist/assets/styles/main.css",
+    "sassBuild": "sass src/assets/styles/main.scss:dist/assets/styles/main.css && npm run cssPrefix",
+    "cssPrefix": "postcss ./dist/assets/styles/main.css --use autoprefixer -d ./dist/assets/styles",
```

В остальных местах пути указываются без точки, может продолжим?

```suggestion
    "cssPrefix": "postcss dist/assets/styles/main.css --use autoprefixer -d dist/assets/styles",
```

---

@pepelsbey сглупил тоже :)